docs/site/Database-migrations.md

```diff
@@ -50,7 +50,7 @@ export class MyModel extends Entity {
 
 For detailed connector-specific settings for defining model schemas and
 auto-migration, check out the specific connector under
-[Database Connectors](Database-connectors.md).
+[Database Connectors](Database-connectors.html).
```

Comment:
Why do we need to use `.html` instead of `.md`?

Reply:
I believe we generally use `.md` instead if the page is part of loopback-next docs, so trying to make it consistent. I could revert the change if it's not intended. 
Also, the link checker would identify it as error because the page doesn't exist, but we may need another solution to bypass this issue.

Reply:
I also tend to use `.md` for consistency.

Reply:
Talked to @raymondfeng, he also thinks we should be consistent and use `.md`.